README.md

```diff
@@ -151,26 +147,8 @@ this.refs.myPortal.openPortal()
 // opens the portal, yay!
 ```
 
-## Contribution
-
-Please, create issues and pull requests.
-
-```shell
-git clone https://github.com/tajo/react-portal
-cd react-portal
-npm install
-npm start
-open http://localhost:3000
-```
-
 **Don't forget to run this before every commit:**
 
 ```
 npm test
 ```
-
```

Comment:
Add this back, something like:

-## Credits
 - [react-portal](https://github.com/tajo/react-portal) originally inspired by the talk [React.js Conf 2015 - Hype!, Ryan Florence](https://www.youtube.com/watch?v=z5e7kWSHWTg) and written by Vojtech Miksu 2015, [miksu.cz](https://miksu.cz), [@vmiksu](https://twitter.com/vmiksu)